python/tk-core/README.md

```diff
@@ -1,4 +1,8 @@
-[![Build Status](https://secure.travis-ci.org/shotgunsoftware/tk-core.png?branch=master)](http://travis-ci.org/shotgunsoftware/tk-core)
```

Comment:
I should have mentionned that we try to avoid including core in the core review since it sometimes clutters it and there are actual files to review both before and after the core files in the middle.